src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -119,9 +119,7 @@ public SonarRunnerBuilder() {
     // all fields are optional
   }
 
-  /**
-   * @deprecated We're moving to using @DataBoundSetter instead and a much leaner @DataBoundConstructor 
-   */
+  // We're moving to using @DataBoundSetter instead and a much leaner @DataBoundConstructor
   @Deprecated
   public SonarRunnerBuilder(String installationName, String sonarScannerName, String project, String properties, String javaOpts, String jdk, String task,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)